protostar/testing/cairo_cheatcodes/declare_contract_cairo_cheatcode.py

```diff
@@ -0,0 +1,102 @@
+import asyncio
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Protocol
+
+from starkware.python.utils import from_bytes
+from starkware.starknet.business_logic.transaction.objects import InternalDeclare
+from starkware.starknet.public.abi import AbiType
+from starkware.starknet.services.api.gateway.transaction import (
+    DEFAULT_DECLARE_SENDER_ADDRESS,
+)
+from starkware.starknet.testing.contract import DeclaredClass
+from starkware.starknet.testing.contract_utils import EventManager, get_abi
+
+from protostar.compiler import ProjectCompiler
+from protostar.testing.cairo_cheatcodes.cairo_cheatcode import CairoCheatcode
+
+
+@dataclass
+class DeclaredContract:
+    class_hash: int
```

Comment:
We should leave it for now, but we may need to provide some more cairo-friendly data here to be able to load it to cairo data structure